packages/google_maps_flutter/google_maps_flutter_ios/ios/google_maps_flutter_ios.podspec

```diff
@@ -22,8 +22,14 @@ Downloaded by pub (not CocoaPods).
   # has been confirmed to be compatible via an example in examples/. See discussion
   # in https://github.com/flutter/flutter/issues/86820 for why this is so broad.
   s.dependency 'GoogleMaps', '< 8.0'
+  s.dependency 'Google-Maps-iOS-Utils', '4.1.0'
```

Comment:
It's probably worth adding a comment here saying that this should be updated once the plugin requires 13.0+.